joint1_controller:
  type: effort_controllers/JointPositionController
  joint: joint1
  pid: 
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5

joint2_controller:
  type: effort_controllers/JointPositionController
  joint: joint2
  pid:
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5

joint3_controller:
  type: effort_controllers/JointPositionController
  joint: joint3
  pid:
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5

joint4_controller:
  type: effort_controllers/JointPositionController
  joint: joint4
  pid:
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5